posix.mak

```diff
@@ -456,13 +467,18 @@ docs.json : ${DMD_REL} ${DRUNTIME_DIR}-${LATEST} \
 	  --only-documented $(MOD_EXCLUDES_PRERELEASE)
 	rm .release-files.txt .release-dummy.html
 
-docs-prerelease.json : ${DMD} ${DRUNTIME_DIR} \
+docs-prerelease.json : ${DMD} ${DMD_DIR} ${DRUNTIME_DIR} \
 		${PHOBOS_DIR} | dpl-docs
+	find ${DMD_DIR}/src -name '*.d' | \
+		sed -e /objc.d/d -e /mscoff/d \
+			-e /objc_glue.d/d $(DMD_EXCLUDE)  \
+		> .prerelease-files.txt
 	find ${DRUNTIME_DIR}/src -name '*.d' | sed -e '/gcstub/d' \
-	  -e /unittest/d > .prerelease-files.txt
+	  -e /unittest/d >> .prerelease-files.txt
 	find ${PHOBOS_DIR} -name '*.d' | sed -e /unittest.d/d \
 	  -e /windows/d >> .prerelease-files.txt
-	${DMD} -c -o- -version=CoreDdoc -version=StdDdoc -Df.prerelease-dummy.html \
+	${DMD} -J$(DMD_DIR)/res -J$(DMD_DIR)/src -c -o- -version=MARS -version=CoreDdoc \
```

Comment:
`MARS` is needed due to [`backend/cc.d`](https://github.com/dlang/dmd/blob/master/src/backend/cc.d#L175). It's a [default flag for all D compilers](https://github.com/dlang/dmd/blob/master/src/posix.mak#L149).